src/Components/DataHistoryExplorer/Internal/TimeSeriesViewer/Internal/TimeSeriesChart/TimeSeriesChart.tsx

```diff
@@ -114,28 +131,41 @@ const TimeSeriesChart: React.FC<ITimeSeriesChartProps> = (props) => {
 
     return (
         <div className={classNames.root}>
-            <ChartCommandBar
-                defaultOptions={chartOptions}
-                onChange={setChartOptions}
-                deeplink={deeplink}
-            />
-            <div className={classNames.chartContainer}>
-                <HighChartsWrapper
-                    seriesData={highChartSeriesData}
-                    isLoading={isLoading}
-                    chartOptions={{
-                        titleAlign: 'left',
-                        legendLayout: 'horizontal',
-                        legendPadding: 0,
-                        hasMultipleAxes:
-                            chartOptions.yAxisType === 'independent',
-                        dataGrouping: chartOptions.aggregationType,
-                        xMinInMillis: xMinDateInMillisRef.current,
-                        xMaxInMillis: xMaxDateInMillisRef.current,
-                        maxLegendHeight: 160
-                    }}
+            {errors.length > 0 ? (
+                <DataHistoryErrorHandlingWrapper
```

Comment:
so the wrapper is a different component that is a sibling to the other stuff? I guess the name `Wrapper` implies that it is like an error boundary and surrounds the child components. If that's not hte case, could we call it "DataHistoryErrorModal/Dialog" or something like that?

Reply:
I did the same thing in ScenePageErrorHandlingWrapper. I think it make sense since it is wrapping the IllustrationMessage component based on the component errors of adapter calls. So there is a specific internal logic to control the IllustrationMessage child component. Does it make sense? I do not have a strong preference though, can change it to "DataHistoryErrorHandlingIllustration"?

Reply:
Yeah. I personally prefer that, but don't feel strongly.